README.md

```diff
@@ -1,7 +1,7 @@
-rnpm link
+rnpm link [![Circle CI](https://img.shields.io/circleci/project/rnpm/rnpm-plugin-link/master.svg)](https://circleci.com/gh/rnpm/rnpm-plugin-link) 
```

Comment:
:D TravisCI was not working for some reason and seems like CircleCI is faster :dancer: